bin/proving-service/.env

```diff
@@ -11,3 +11,10 @@ MPS_AVAILABLE_WORKERS_POLLING_INTERVAL_MS="20"
 MPS_HEALTH_CHECK_INTERVAL_SECS="1"
 MPS_ENABLE_METRICS="false"
 RUST_LOG="info"
+
+# Worker configuration
+MPS_WORKER_HOST="0.0.0.0"
```

Comment:
Question: in what circumstances would it make sense to set worker host to something other than `0.0.0.0`? Basically, I'm wondering if the same logic we used to get rid of the proxy host would apply here as well.

Reply:
Unlike the proxy, I think of the Worker as a service that can be started independently by any user, and in that case, for security reasons, the user could switch between 127.0.0.1 and 0.0.0.0

Reply:
Are `127.0.0.1` and `0.0.0.0` the only two practical settings for this parameter? If so, maybe these should be replaced by an enum which explicitly identifies the use case?

I don't actually know too much about this - but why would we sometimes chose `127.0.0.1` and other times `0.0.0.0`?